Jenkinsfile

```diff
@@ -1,48 +1,84 @@
-node {
-	timeout(60) {
-		timestamps {
-			properties([
-				[$class: 'BuildDiscarderProperty', strategy: [$class: 'LogRotator', numToKeepStr: '15']],
-				pipelineTriggers([cron('H 2 * * *')])
-			])
-			
-			stage('Checkout') {
-				sh '''
-					if [ -d ".git" ]; then
```

Comment:
is this no longer needed?

Reply:
Another thing where I wonder why this was ever there. I'll discuss that with @Hejado

Reply:
@kthoms any news on this ?

Reply:
@cdietrich I have no idea, why it was added in the first place, and there are certainly prettier ways to make sure our workspace is clean. @kthoms added it again, because we think it's not extremely necessary to remove now. On JIRO however, this will be irrelevant.

Reply:
:+1: but then we should leave it here to be on the safe side